test/system/700-play.bats

```diff
@@ -455,36 +455,21 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
 }
 
-@test "podman play --annotation > Max" {
-    TESTDIR=$PODMAN_TMPDIR/testdir
-    RANDOMSTRING=$(random_string 65)
-    mkdir -p $TESTDIR
-    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
-    run_podman 125 play kube --annotation "name=$RANDOMSTRING" $PODMAN_TMPDIR/test.yaml
-    assert "$output" =~ "annotation exceeds maximum size, 63, of kubernetes annotation:" "Expected to fail with Length greater than 63"
-}
-
-@test "podman play --no-trunc --annotation > Max" {
-    TESTDIR=$PODMAN_TMPDIR/testdir
-    RANDOMSTRING=$(random_string 65)
-    mkdir -p $TESTDIR
-    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
-    run_podman play kube --no-trunc --annotation "name=$RANDOMSTRING" $PODMAN_TMPDIR/test.yaml
-}
-
-@test "podman play Yaml with annotation > Max" {
-   RANDOMSTRING=$(random_string 65)
+@test "podman play Yaml with annotation size beyond limits" {
```

Comment:
I would honestly consider dropping max-size tests. We don't validate Podman against 256kb strings anywhere else, and it seems like it has the potential to be a slow test.

Reply:
We should test the deprecated flag, though.